demo/spec/components/previews/checkbox_group_preview.rb

```diff
@@ -87,4 +96,11 @@ def additional_attribute_checkboxes
       { label: "Option 2", value: "2", "data-testid": "second-input" }
     ]
   end
+
+  def custom_id_checkboxes
+    [
+      { label: "Option 1", value: "1", id: "test-id-first" },
```

Comment:
Isn't the `id` defined as a parameter on the component itself rather in the options array?

i.e.

```ruby
CitizensAdviceComponents::CheckboxGroup.new(
  name: "example_form[contact_preferences][]",
  id: "example_form_contact_preferences"
)
```

Reply:
Yeah, good shout, thanks! I'll adjust that